appmap/test/data/pytest/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.9
+
+RUN pip install poetry
```

Comment:
```suggestion
RUN pip install poetry cryptography_vectors
```

Reply:
This isn't necessary with the full image AFAICT.

Reply:
@dividedmind ran into this issue with `python:3.9` image.

Reply:
Weird, works for me. Oh well, I suppose it can't hurt, right?

Reply:
I just pulled the latest version of the `python:3.9` image. `pip install poetry` ran fine in it, without needing to specify `cryptography_vectors`.

@virajkanwade Can you pull the image and see if you're able to reproduce this problem, please? If you can, it woul dbe good to figure out why.

Reply:
I believe you have removed docker dependency.

Altough 2 things:
1. `removeprefix` and `removesuffix` functions are 3.9 feature. So we should update the min py version check to `(3, 9) `
2. For some reason, for me, the test fails as the generated appmap has `classMap` empty and `events` empty.

Reply:
Sorry, I added https://github.com/applandinc/appmap-python/pull/31#issuecomment-779471319 before I'd seen this.

1. I'll fix the version check.

2. Yes, I removed the dependency on Docker. Re the empty AppMap, make sure that the version of pytest.ini you've got has these lines in it.

```
# running in a subprocess ensures that environment variables are set
# correctly and no classes are loaded.
addopts = --runpytest subprocess
```

If it doesn't, please retry with the latest commit. If it does, let me know and I'll try to figure out why.